app/classifier/tasks/survey/editor.cjsx

```diff
@@ -159,10 +247,10 @@ module.exports = React.createClass
 
       {if @props.task.questionsOrder?.length
         <div>
-          <Details summary="Questions">
+          <Details summary={<span className="details">Questions</span>}>
```

Comment:
i could be wrong, but unless Details uses __dangerouslySetInnerHtml i don't think this will have the intended effect. (not necessary for merge)